src/Adapter/MSTest.CoreAdapter/MSTestSettings.cs

```diff
@@ -338,6 +344,26 @@ private static MSTestSettings ToSettings(XmlReader reader)
                                 break;
                             }
 
+                        case "CLASSCLEANUPLIFECYCLE":
+                            {
+                                var value = reader.ReadInnerXml();
+                                if (Enum.TryParse(value, out ClassCleanupLifecycle lifecycle))
+                                {
+                                    settings.ClassCleanupLifecycle = lifecycle;
+                                }
+                                else
+                                {
+                                    throw new AdapterSettingsException(
+                                        string.Format(
+                                            CultureInfo.CurrentCulture,
+                                            Resource.InvalidParallelScopeValue,
```

Comment:
This uses an incorrect resource.
```suggestion
                                            Resource.InvalidClassCleanupLifecycleValue,
```